06_modules_lab/01.py

```diff
@@ -1,4 +1,17 @@
-""" Write a program that takes a count
-from sys.argv import and prints "Hello Python"
-count times
-"""
+"""
+      _      _      _
+   __(.)< __(.)> __(.)=
+   \___)  \___)  \___)   
+
+Python course - Verint
+Week 1
+
+"""
+
+import sys
+
+i = int(sys.argv[1])
+
+while i > 0:

```

Comment:
A for loop is preferred when number of iterations is known in advance (`for i in range(...)`)